docs/tutorials/irradiance.ipynb

```diff
@@ -367,7 +367,7 @@
    "source": [
     "See the online documentation for clear sky modeling examples.\n",
     "\n",
-    "http://pvlib-python.readthedocs.io/en/latest/clearsky.html\n",
+    "http://pvlib-python.readthedocs.io/en/stable/user_guide/clearsky.html\n",
```

Comment:
The link seems to be broken. Perhaps it should be: https://pvlib-python.readthedocs.io/en/stable/clearsky.html

Reply:
That's actually intentional -- the problem is that `user_guide` doesn't currently exist on `stable` (only on `latest`), but as soon as we release 0.9.1 this new link should work.  Not ideal, but I figure it's ok for that link to be broken until the next release.